### ISSUE TITLE
can not control voice character

### ISSUE THREAD

**Issue by ggsonic** [THUDM/GLM-4-Voice#38]
ask to speak in male /female voice, in Cantonese or other ones, no effect, always in same female voice

**Comment by ggsonic** [comment]
ok now

**Comment by BlcDec** [comment]
> ok now

Hello, how did you solve it?

**Comment by kssmmm** [comment]
I got the same problem，and I tried out some other female voices, and there might be a slight change

**Comment by BlcDec** [comment]
> I got the same problem，and I tried out some other female voices, and there might be a slight change

Yes, there is a slight variation, so this model can only output a female's style?

**Comment by xiaohanghu** [comment]
I think it's because the decoder is trained by female's voice, am I right?

**Comment by xiaoyuanzi22333** [comment]
same issue, I tried to change the condition and speaker embedding inside the flow model, but it doesn't work. Does this mean the model can only output the same female voice?
